.circleci/config.yml

```diff
@@ -19,21 +19,51 @@ commands:
   check-uncommitted:
     description: Check for uncommitted changes.
     steps:
-      - run: 
-          name: Check for uncomitted changes using git porcelain command.
+      - run:
+          name: Check for uncomitted changes.
           command: |
             if [[ `git status --porcelain` ]]; then
               echo "ERROR: uncommitted changes"
               git status --porcelain
               exit 1
             fi
+  install-golangci-lint:
+    description: Install golangci-lint
+    parameters:
+      version:
+        type: string
+        default: 1.21.0
+      gobin:
+        type: string
+        default: /go/bin
+      prefix:
+        type: string
+        default: v1
+        description: Prefix for cache key to store the binary.
+    steps:
+      - restore_cache:
```

Comment:
I really like the caching

Reply:
Me too! I note also that /not/ having multiple CircleCI steps saves us the cost of restoring the cache each time.